templates/kapeta/block-type-cli/index.ts.hbs

```diff
@@ -12,7 +12,7 @@ program
     });
 
 // Catch all command to show a custom message for unknown commands
-program.command('*', { noHelp: true }).action(() => {});
+program.command('*', { hidden: true }).action(() => {});
```

Comment:
Is this due to updating the CLI package?

Reply:
No, it was just me who used `noHelp` in my first iteration of the CLI language target. But I have now found out it is deprecated and `hidden` is the new way of hiding a command.